app/src/main/java/app/attestation/auditor/QRScannerActivity.kt

```diff
@@ -0,0 +1,85 @@
+package app.attestation.auditor
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.util.Size
+import androidx.appcompat.app.AppCompatActivity
+import androidx.camera.core.CameraSelector
+import androidx.camera.core.ImageAnalysis
+import androidx.camera.core.Preview
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.camera.view.LifecycleCameraController
+import androidx.camera.view.PreviewView
+import androidx.core.content.ContextCompat
+
+class QRScannerActivity : AppCompatActivity() {
+
+    public override fun onCreate(state: Bundle?) {
+        super.onCreate(state)
+        setContentView(R.layout.activity_qrscanner)
+        startCamera()
+    }
+
+
+    private fun startCamera(){
```

Comment:
Minor style nit: should have a space between `)` and `{`.

Reply:
fixed.